Драфт схемы БД с патчем

Artem-7 opened the pull request:
Первый вариант схемы (тот, что готовил к презентации в пятницу). На текущий момент не учитывает требование хранить общие для маршрута настройки отдельно от варианта

Commit:
Проект структуры МС (таблицы)

Commit:
Патч на создание таблиц схемы

rnljul reviewed (commented):
Ознакомился со структурой.

rnljul reviewed (changes requested):
надо кодировку править

chameleon82 reviewed (approved):
С кодировкой беда. Давайте в UTF8 все вести?

chameleon82 commented:
Рома, ты можешь заапрувить?

rnljul commented:
Я не вижу кнопку APPROVE, и у меня нет прав на запись, значит сливать тоже не могу.
На счет кодировки - давайте UTF-8.

rnljul commented:
С аппрувом разобрался.

Commit:
Исправление кодировки файлов ANSI->UTF-8

Artem-7 commented:
Добавил в реквест коммит, исправляющий кодировку

rnljul reviewed (commented):
Давайте заменим префикс MV на RS (route system)

Artem-7 committed:
1. Переименование объектов. Изменил префиксы с MW на RS
2. Патч сейчас в двух вариантах. Первая версия накатывает схему с нуля, вторая версия - переименовывает существующий объекты. Выбираем то, что нравится

rnljul commented:
А у кого права на запись есть? Слейте реквест!
